modules/seo-tools/jetpack-seo.php

```diff
@@ -0,0 +1,156 @@
+				$tags['og:title'] = $custom_title;
+			}
+
+			if ( !empty ( $post_custom_description ) ) {
+				$tags['og:description'] = $post_custom_description;
+			}
+
+			return $tags;
+		} );
+	}
+
+	private function get_authors() {
+		global $wp_query;
+
+		$authors = array();
+
+		foreach ( $wp_query->posts as $post ) {
+			$authors[] = get_the_author_meta( 'display_name', (int) $post->post_author );
+		}
+
+		$authors = array_unique( $authors );
+
+		return $authors;
+	}
+
+	function meta_tags() {
+		global $wp_query;
+
+		$period = $template = '';
+		$meta = array();
+		$conflicted_themes = apply_filters( 'jetpack_seo_meta_tags_conflicted_themes', array() );
+
+		if ( isset( $conflicted_themes[ get_option( 'template' ) ] ) ) {
+			return;
+		}
+
+		$front_page_meta = Jetpack_SEO_Utils::get_front_page_meta_description();
+		$description = $front_page_meta ? $front_page_meta : get_bloginfo( 'description' );
+
+		$site_host = apply_filters( 'jetpack_seo_site_host', 'WordPress' );
+
+
+		$meta['title'] = sprintf( _x( '%1$s on %2$s', 'Site Title on WordPress', 'jetpack' ), get_bloginfo( 'title' ),	$site_host );
+		$meta['description'] = trim( $description );
+
+		// Try to target things if we're on a "specific" page of any kind.
+		if ( is_singular() ) {
+			$meta['title'] = sprintf( _x( '%1$s | %2$s', 'Post Title | Site Title on WordPress', 'jetpack' ), get_the_title(), $meta['title'] );
+
+			// Business users can overwrite the description.
+			if ( ! ( is_front_page() && Jetpack_SEO_Utils::get_front_page_meta_description() ) ) {
+				$description = Jetpack_SEO_Posts::get_post_description( get_post() );
+
+				if ( $description ) {
+					$description = wp_trim_words( strip_shortcodes( wp_kses( $description, array() ) ) );
+					$meta['description'] = $description;
+				}
+			}
+
+		} else if ( is_author() ) {
+			$obj                 = get_queried_object();
+			$meta['title']       = sprintf( _x( 'Posts by %1$s | %2$s', 'Posts by Author Name | Blog Title on WordPress', 'jetpack' ), $obj->display_name, $meta['title'] );
+			$meta['description'] = sprintf( _x( 'Read all of the posts by %1$s on %2$s', 'Read all of the posts by Author Name on Blog Title', 'jetpack' ), $obj->display_name, get_bloginfo( 'title' ) );
+		} else if ( is_tag() || is_category() || is_tax() ) {
+			$obj = get_queried_object();
+
+			$meta['title'] = sprintf( _x( 'Posts about %1$s on %2$s', 'Posts about Category on Blog Title', 'jetpack' ), single_term_title( '', false ), get_bloginfo( 'title' ) );
+
+			$description = get_term_field( 'description', $obj->term_id, $obj->taxonomy, 'raw' );
+			if ( ! is_wp_error( $description ) && '' != $description ) {
+				$meta['description'] = wp_trim_words( $description );
+			} else {
+
+				$authors             = $this->get_authors();
+				$meta['description'] = wp_sprintf( _x( 'Posts about %1$s written by %2$l', 'Posts about Category written by John and Bob', 'jetpack' ), single_term_title( '', false ), $authors );
```

Comment:
That was on purpose since `$authors` is an array. :) I've used it to replace the deprecated `oxford_comma` that is currently used on dotcom side, and doesn't exist in Jetpack. There is more info about it in contributor notes here: https://developer.wordpress.org/reference/functions/wp_sprintf/